### PR TITLE
Update license files

### PR DESCRIPTION
**Pull request by rugeGerritsen**
This aligns all copyright files and bumps them to 2020

### CODE REVIEW

File: mpsl/license.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright (c) 2018 - 2020 Nordic Semiconductor ASA
```
Comment (tejlmand) [inline mpsl/license.txt]: Just noted that this license states that it is copyrighted 2018 - 2020.
But the old license, which is now removed, states 2007 - 2019.

I assume, this means the code was originally copyrighted in 2007, and thus I assume the start year should still be 2007.

Or should the start year be the year we changes the license, cause in that case, it should probably just state 2020, as the years 2007 - 2019 was under old license.

2018 - 2020, at least seems incorrect to me.
(Personally I assume 2007 - 2020 to be most correct)

But I'm not a lawyer, so unsure of what is most correct.
Could you check up on this, and also explain why 2018 is start year :)

File: ble_controller/license.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright (c) 2018 - 2020 Nordic Semiconductor ASA
```
Comment (tejlmand) [inline ble_controller/license.txt]: Same comment regarding the years as in mpsl/license.txt

File: mpsl/license.txt (hunk 1 above)
Comment (carlescufi) [reply mpsl/license.txt]: I actually don't know either. I think the correct format is always `<start year> - <end year>` but I doubt that MPSL was introduced 13 years ago?

Comment (tejlmand) [reply mpsl/license.txt]: > but I doubt that MPSL was introduced 13 years ago?

Now that you mention it, I highly doubt that too for mpsl.

Hope that @rugeGerritsen will provide more details on the correct years for this, as well as for the ble controller. The ble controller could actually contain code that originates a long time back.

Comment (tejlmand) [reply mpsl/license.txt]: @rugeGerritsen any updates, been more than a week :wink:

**Pull request closed without merging**